templates/emails/group.expense.created.hbs

```diff
@@ -26,6 +26,12 @@ Subject: New expense on {{group.name}}: {{{currency expense.amount currency=expe
 <center>
   <h1>{{{currency expense.amount currency=expense.currency}}}</h1>
   <div>{{expense.title}}</div>
+  <div>From: {{user.name}}</div>
+  {{#if user.paypalEmail}}
+  <div>Paypal Email: {{user.paypalEmail}}</div>
+  {{else}}
+  <div>Paypal Email: {{user.email}}</div>
+  {{/if}}
```

Comment:
It's better to limit the logic in the templates. This should be taken care of before sending the data to the template (`user.paypalEmail = user.paypalEmail || user.email`)